parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -241,6 +242,27 @@ configureCNIIPTables() {
     fi
 }
 
+configureCNINFTables() {
+    if [[ "${IS_IPV6}" = "true" ]]; then
+        # Delete the table in a subshell so that we can eat the failed return code
+        (nft -na -- list table ip6 azureSLBProbe >/dev/null 2>&1 && nft -- delete table ip6 azureSLBProbe; exit 0)
+
+        # Create a custom table
+        nft -- add table ip6 azureSLBProbe
```

Comment:
would it make sense to have the full table/chain defined in a file and just `nft -f $file` ? I would fine that a little more readable personally